images/rhel/rhel8/ubi8-toolbox/extra-packages

```diff
@@ -0,0 +1,50 @@
+bzip2
+diffutils
+dnf-plugins-core
+git
+gnupg2-smime
+hostname
+iputils
+less
+lsof
+man-db
+openssh-clients
+pigz
+procps-ng
+rsync
+sudo
+time
+unzip
+vte-profile
+wget
+xz
+zip
+python3
+alternatives
+avahi-libs
+coreutils-common
+dejavu-sans-fonts
+jansson
+libICE
+libSM
+libX11
+libX11-common
+libXau
+libXext
+libXmu
+libXt
+libevent
+libgomp
+libpkgconf
+libxcb
+nano
```

Comment:
Is `nano` really the default on most RHEL 8 installations?

Reply:
I think that vi/vim is the default editor in RHEL 8.